source/sass/buyers-guide/views/catalog.scss

```diff
@@ -35,6 +35,10 @@ body.catalog {
         padding-top: 90px;
         padding-bottom: 90px;
       }
+
+      .dark-theme {
+        color: $white;
+      }
```

Comment:
I tried using this, but that didn't want to work:
```css
      h1 {
        @include text-base-style("Zilla Slab", 300, $black, $white);
      }

      p {
        @include text-base-style("Nunito Sans", 300, $black, $white);
      }
```

Reply:
We shouldn't need any additional SCSS here. Having `dark-theme` on `intro-text` should just get the right styling to kick in! 😄

Reply:
```suggestion
```